apps/api/src/app/models/students/student.service.ts

```diff
@@ -19,4 +20,7 @@ export class StudentModelService {
   async update(data: StudentUpdated) {
     await this.model.findOneAndUpdate({ _id: Types.ObjectId(data.id) }, { $set: data.student })
   }
+  async getById(id: string): Promise<CreateStudentPayloadDto> {
+    return this.model.findById({ _id: Types.ObjectId(id) })
```

Comment:
Why `this.model` is public?

Reply:
> Why `this.model` is public?

model should be public, because sometimes you need use it directly from handler